templates/appliance/community.html

```diff
@@ -1,216 +1,213 @@
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
 
 {% block content %}
-<section class="p-strip--suru-topped">
-  <div class="row u-equal-height">
-    <div class="col-8">
-      <h1>
-        Welcome to our Ubuntu Appliance community
-      </h1>
-      <p>
-        Whether you are a power user, a developer, a member of the Ubuntu project or a new contributor to Ubuntu Appliances, we invite you to join our Ubuntu Appliance Discourse and make your voice heard.
-      </p>
+
+  {% call(slot) vf_hero(
+    title_text='Welcome to our<br /> Ubuntu appliance community',
+    layout='50/50'
+    ) -%}
+    {%- if slot == 'description' -%}
       <p>
-        <a href="https://discourse.ubuntu.com/c/appliance">Join the Ubuntu Appliance Discourse</a>
+        Whether you are a power user, a developer, a member of the Ubuntu project or a new contributor to Ubuntu appliances, we invite you to join our Ubuntu appliance Discourse and make your voice heard.
       </p>
+    {%- endif -%}
+    {%- if slot == 'cta' -%}
+      <a href="https://discourse.ubuntu.com/c/appliance">Join our Ubuntu appliance Discourse&nbsp;&rsaquo;</a>
+    {%- endif -%}
+    {%- if slot == 'image' -%}
+      <div class="p-image-container--3-2 is-cover">
+        {{ image(url="https://assets.ubuntu.com/v1/da070994-hero image.png",
+                alt="",
+                width="1800",
+                height="1128",
+                hi_def=True,
+                loading="auto",
+                attrs={"class": "p-image-container__image"},) | safe
+        }}
+      </div>
+    {%- endif -%}
+  {% endcall -%}
+
+  <section class="p-section">
+    <hr class="is-fixed-width p-rule" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>
+          New Ubuntu appliance ideas
+          <br />
+          and proposals
+        </h2>
+      </div>
+      <div class="col">
+        <p>To suggest a new Ubuntu appliance:</p>
+        <hr class="p-rule--muted" />
+        <ol class="p-list--divided u-no-margin--bottom">
+          <li class="p-list__item">
+            <a href="https://discourse.ubuntu.com/c/appliance">Share your idea</a> with a <a href="https://discourse.ubuntu.com/c/appliances/proposed-new-appliances">Proposed New appliance</a> topic to get community feedback
+          </li>
+          <li class="p-list__item">
+            Publish your appliance software at <a href="https://snapcraft.io/docs/getting-started">snapcraft.io</a>
+          </li>
+          <li class="p-list__item">
+            Test on a <a href="/download/intel-nuc">PC</a> or <a href="/download/raspberry-pi-core">Raspberry Pi</a> with a <a href="/core/docs/try-pre-built-images">pre-built Ubuntu Core image</a>
+          </li>
+          <li class="p-list__item">Optimize your appliance images for devices you want to support</li>
+        </ol>
+      </div>
```

Comment:
Copy doc and figma are slightly out of sync here, left a [comment](https://docs.google.com/document/d/1DpU0AzAJ4gBa_au4gELBUD3vLz8l-FnAwFdERBHlvwc/edit?disco=AAABdMHvWOw) on the doc